src/test/unit/pg_unittest.cc

```diff
@@ -35,9 +35,25 @@ extern "C" {
 PG_REGISTER_WITH_RESET_TEMPLATE(motorConfig_t, motorConfig, PG_MOTOR_CONFIG, 1);
 
 PG_RESET_TEMPLATE(motorConfig_t, motorConfig,
-    .dev = {.motorPwmRate = 400},
+    .dev = {
+        .motorPwmRate = 400,
+        .motorPwmProtocol = 0,
```

Comment:
this is too cumbersome for unittest. It is imo better to disable the warning. Or move stubs to .c file

Reply:
- clang now expects all members to be present once a member is referenced.
- not sure how to implement your suggestion.

Reply:
If it removes the complexity, it would be better to do as @ledvinap suggests and disable the warning for the unit test only.

The goal of the unit test is to test our code interface and logic, not the GCC warnings for the unit tests themselves.

Reply:
Agree, but rather suppressing the warning need to know (have an example) how to implement the member into a stub as suggested.